audio/miles_adlib.cpp

```diff
@@ -270,7 +270,7 @@ class MidiDriver_Miles_AdLib : public MidiDriver {
 };
 
 MidiDriver_Miles_AdLib::MidiDriver_Miles_AdLib(InstrumentEntry *instrumentTablePtr, uint16 instrumentTableCount)
-	: _masterVolume(15), _opl(0),
+	: _masterVolume(255), _opl(0),
```

Comment:
Were other engines which use this driver tested with this change? Why this change is required at all?